src/Compiler/Compiler.cpp

```diff
@@ -376,12 +376,12 @@ bool Compiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
     }
     std::string actualType = "VERILOG_2001";
     Design::Language language = Design::Language::VERILOG_2001;
-    const std::string file = argv[1];
-    if (strstr(file.c_str(), ".vhd")) {
+    auto file = QString(argv[1]).toLower();
+    if (file.contains(".vhd")) {
```

Comment:
need to check if is the only place that have this problem.

Reply:
https://github.com/os-fpga/FOEDAG/blob/5d94adc66230aeade97d55630bbc61aa5a881963/src/Compiler/Compiler.cpp#L492

Reply:
covered in second commit. also for .eblif